src/main/java/core/basesyntax/Plane.java

```diff
@@ -1,5 +1,86 @@
 package core.basesyntax;
 
 public class Plane {
+    private String model;
+    private String color;
+    private double maxSpeed;
+    private boolean propeller;
+    private boolean airframe;
+    private boolean wings;
```

Comment:
```suggestion
    private boolean hasWings;
```
https://wiki.eclipse.org/Recommenders/CodingConventions#:~:text=is%20prefix%20should%20be%20used%20for%20boolean%20variables%20and%20methods.,-isSet%2C%20isVisible%2C%20isFinished&text=This%20is%20the%20naming%20convention,names%20like%20status%20or%20flag.